src/base/system_parameter.c

```diff
@@ -6346,6 +6351,19 @@ static SYSPRM_PARAM prm_Def[] = {
    (char *) NULL,
    (DUP_PRM_FUNC) NULL,
    (DUP_PRM_FUNC) NULL},
+  {PRM_ID_ER_LOG_PTS_ATOMIC_REPL_DEBUG,
```

Comment:
I want to be able to activate de-activate this debug logging without changes to the binary.
Is there a way to do this without a system param?
The param should be removed as soon as we are confident that atomic replication is realiable. There is a similar param `PRM_ID_ER_LOG_PTS_REPL_DEBUG` only used for MVCC replication (although the name is misleading). I will created a patch to change that name as well.
Actually, maybe my mistake is that I guarded the usage with `NDEBUG`.
I will remove the NDEBUG guards then because the logging might prove useful in release runs as well.